style.css

```
@@ -555,6 +555,64 @@ header {
 }
 
 /*---footer---*/
+footer {
+  padding-top: 50px;
+}
+
+footer .ft-logo {
+  margin-bottom: 30px;
+  text-align: center;
+}
+
+footer .ft-logo a {
+  display: inline-block;
+}
+
+footer .ft-logo a img {
+  width: auto;
+  height: 25px;
+}
+
+footer .ft-info {
+  text-align: center;
+  line-height: 2;
+  margin-bottom: 50px;
+  padding: 0 20px;
+}
+
+footer .ft-nav {
+  background-color: #f5f5f5;
+  border-top: 1px solid #dddddd;
+  border-bottom: 1px solid #dddddd;
+}
+
+footer .ft-nav .container-fluid .row > div {

```

(以下の行のものも全て)divじゃなくて`.ft-nav-link`とかの方が良いのかなと(必ず直下にdivがあることが明らかな場合なら良いですが！)

また、`:not`擬似要素を使うことで594行目の`borde: none`のためのcssが消せますね
```suggestion
footer .ft-nav .container-fluid .row > .ft-nav-link:not(:last-of-type) {
```